src/Components/Blueprints/BlueprintActionsMenu.tsx

```diff
@@ -60,11 +71,36 @@ export const BlueprintActionsMenu: React.FunctionComponent<
         >
           Edit details
         </DropdownItem>
-        <DropdownItem>Download blueprint (.json)</DropdownItem>
+        <DropdownItem
+          onClick={() => {
+            trigger({ id: selectedBlueprintId || '' });
+          }}
+        >
+          Download blueprint (.json) <BetaLabel />
+        </DropdownItem>
         <DropdownItem onClick={() => setShowDeleteModal(true)}>
           Delete blueprint
         </DropdownItem>
       </DropdownList>
     </Dropdown>
   );
 };
+
+async function handleExportBlueprint(
```

Comment:
Alright, as per discussion on Slack, this situation is different enough that the way you've done it here is better. Namely, we don't have to trigger a lazyQuery in an onClick handler when using `<a>` since the data is already available.